grafana/Dockerfile

```diff
@@ -23,10 +23,10 @@ ENV NODE_ENV production
 RUN yarn build
 
 RUN go mod verify
-RUN make build-go
+RUN make build-go # GO_BUILD_TAGS=${GO_BUILD_TAGS} WIRE_TAGS=${WIRE_TAGS}
```

Comment:
Just Question.
What's the purpose of this comment.
I can't find description like this in upstream build procedure.

Reply:
Thank you. I forgot to check this tags which seems be added v9.5.0
https://github.com/grafana/grafana/blob/v9.5.0/Dockerfile#L67

Reply:
Thank you for your answer. I understand.
I was looking the v9.4.0 Dockefile.

Reply:
I confirmed it. We need not set these options, so I removed them.